Podverse/PodcastsTableViewController.swift

```diff
@@ -223,35 +221,16 @@ extension PodcastsTableViewController:UITableViewDelegate, UITableViewDataSource
     
     func tableView(_ tableView: UITableView, editActionsForRowAt indexPath: IndexPath) -> [UITableViewRowAction]? {
         let podcastToEdit = subscribedPodcastsArray[indexPath.row]
-        var subscribeOrFollow = "Subscribe"
-        
-        let subscribeOrFollowAction = UITableViewRowAction(style: .default, title: subscribeOrFollow, handler: {action, indexpath in
-            if subscribeOrFollow == "Subscribe" {
-                //PVSubscriber.subscribeToPodcast(podcastToEdit.feedUrl, podcastTableDelegate: self)
-            } else {
-                //PVFollower.followPodcast(podcastToEdit.feedUrl, podcastTableDelegate: self)
-            }
-        })
-        
-        subscribeOrFollowAction.backgroundColor = UIColor(red: 0.5, green: 0.5, blue: 0.5, alpha: 1.0);
         
         let deleteAction = UITableViewRowAction(style: .default, title: "Delete", handler: {action, indexpath in
-            
-            // Remove Player button if the now playing episode was one of the podcast's episodes
-//            if let nowPlayingEpisode = PVMediaPlayer.shared.currentlyPlayingItem {
-////                if podcastToEdit.episodes.contains(nowPlayingEpisode) {
-////                    self.navigationItem.rightBarButtonItem = nil
-////                }
-//            }
             self.subscribedPodcastsArray.remove(at: indexPath.row)
             self.tableView.deleteRows(at: [indexPath], with: .fade)
-            
-            //PVFollower.unfollowPodcast(podcastToEdit.objectID, completionBlock: nil)
-            
-            //self.showFindAPodcastIfNoneAreFollowed()
+            DispatchQueue.global().async {
```

Comment:
Suggestion, add the Dispatch.Global inside the deletePodcast method so you wont have to do it every time you call it :)